src/content/releases/v0.4.0.mdx

```diff
@@ -0,0 +1,117 @@
+---
+title: "Docs & Files, New Homepage, Redesigned Space Dashboard, and more"
+version: "0.4"
+date: 2025-02-03
+description: "Operately v0.4 introduces Docs & Files, simplifies the homepage and space dashboard, unifies Goals & Projects, and introduces new roles for company administration."
+published: true
+---
+
+import BigImage from '../../components/BigImage.astro';
+import docsFiles from './assets/v040/docs-files.png';
+import homepage from './assets/v040/homepage.png';
+import spaceDashboard from './assets/v040/space-dashboard.png';
+import unifiedGoalsAndProjects from './assets/v040/unified-goals-and-projects.png';
+import adminsOwners from './assets/v040/admins-owners.png';
+import drafts from './assets/v040/drafts.png';
+
+Operately v0.4 is here, packed with improvements that make your Operately
+experience even better. In this release, we're focusing on document management, 
+improving the homepage and space dashboards, and introducing new roles for 
+account administration.
+
+## Docs & Files: A central hub for all your work documents
+
+Every space now has a dedicated Docs & Files section. Upload, organizate, and
+find documents right where your team is working. No more searching through email
+threads or Slack channels to find that one document you need. Documents live
+where the work happens.
+
+<BigImage src={docsFiles} alt="Docs & Files" />
+
+## Redesigned homepage
+
+We've rebuilt the homepage for clarity. You'll see all your spaces laid out
+with the people who are part of them.
+
+On the bottom of the page, you'll find a feed of all the latest updates from
+your spaces, so you can stay up-to-date with what's happening in your company.
+
+<BigImage src={homepage} alt="New Homepage" />
+
+## New space dashboard
+
+We’ve redesigned the space dashboard to give you a better overview of what’s
+happening in your space. The new dashboard provides a snapshot of all the
+projects, goals, and discussions in the space, so you can quickly see what’s
+going on and where you need to focus your attention.
+
+<BigImage src={spaceDashboard} alt="New Space Dashboard" />
+
+## Unified Goals & Projects
```

Comment:
We may say something like "Goal Map: See all goals and projects at a glance". Or perhaps we're not yet mentioning the term "Goal Map" intentionally because it may evolve further?

Reply:
Yeah, I avoided the word Goal Map, not yet sure if we will stick with it or something else like "Work map". Let's introduce it in the next release, not this one.